reference_guide/api_changes_list_2021.md

```diff
@@ -87,6 +87,15 @@ Please see [Incompatible API Changes](api_changes_list.md) on how to verify comp
 `com.intellij.openapi.fileEditor.impl.EditorTabTitleProvider.getEditorTabTitle(Project, VirtualFile, EditorWindow)` method parameter `EditorWindow` removed
 : This parameter never needed, but lead to code coupling
 
+`intelij.platform.util.strings` package removed
```

Comment:
`intelij.platform.util.strings` is name of the module, not the package. As far as I know, package names weren't changes, classes were just moved to a different module. So this change doesn't affect regular plugin developers, it may affect only those who add direct dependencies on [Maven artifacts](https://plugins.jetbrains.com/docs/intellij/intellij-artifacts.html) generated from intellij modules. If we want to document that change (I doubt that we should), it's better to describe it in terms of Maven IDs: artifact "com.jetbrains.intellij.platform:util-strings" is replaced by "com.jetbrains.intellij.platform:util-base".

Reply:
Got it, reverting.